robottelo/config/validators.py

```diff
@@ -197,6 +197,30 @@
         ),
         Validator("report_portal.fail_threshold", default=20),
     ],
+    repos=[
+        Validator(
+            'repos.rhel6_repo',
+            'repos.rhel7_repo',
+            'repos.rhel8_repo',
```

Comment:
Add `repos.rhel_repo_host` the most important one :)

or are you leaving it upto the user to fill out the URLs directly instead of depending on repos.rhel_repo_host ?

Reply:
Think about it this way - there is formatting in the example, but the only fields that really need validated are the actual constructed repo URLs.  Tests/framework doesn't actually use `repos.rhel_repo_host`, and if its missing you'll get an immediate KeyError when trying to load settings.